enyojs/ares-project
# ENYO-2788: A build status Widget is added in the Phonegap build panel.

This widget contains platforms icon-buttons. By clicking on an icon, a status message is shown .
The request to get a new build status is fired when the value of the selected AppId change.

How to test: 
1- Open the panel "Phonegap Build"
2- If the value selected in the AppId picker is "New Application" then all the platforms icons colors must be "Gray", by clicking on any icon a message must appear under this widget saying "Build the application first".
3- If the value selected in the AppId picker is an appId, the the icons colors must change depending on the status message : 
- The build is complete : color green, message: "Download link: <url to download the application>"
- The build is pending : color gray, message "Build in progress"
- The build failed : color: red, message: "Error: <error message>"

Ready for review.

Tested on Chrome, FireFox,  (Windows 7)
Enyo-DCO-1.1-Signed-off-by: Rafik Adiche mahmoud-rafik.adiche@hp.com
